android-resources/www/index.html

```diff
@@ -0,0 +1,99 @@
+<!DOCTYPE html>
+<html>
+  <script src="https://cdn.polyfill.io/v2/polyfill.min.js?features=default,Array.prototype.find,Number.isFinite,Number.isInteger,console,console.log,document.head,performance.now"></script>
+  <head>
+    <title>CodeWorld</title>
```

Comment:
Does the title matter?  If so, it should probably be set to the app name.  If it doesn't, take it out?